subscription-engine/kafka-ws/Dockerfile

```diff
@@ -0,0 +1,35 @@
+FROM node:10.16.3-alpine AS base
```

Comment:
I don't know, maybe I'm wrong about having to be version 10 of the node. 
I think it may only be valid for our old library.
I will check

Reply:
https://github.com/dojot/dojot/pull/1436#discussion_r406455021